### DISCUSSION TITLE
EKS linkerd viz check fail. FailedDiscoveryCheck: failing or missing response

### DISCUSSION THREAD

**Discussion by dennyvettom** [linkerd/linkerd2#12039] [Help]
I am installing Linkerd and viz extension on EKS  1.28, and facing issues with Linkerd Viz checks.   APIService linkerd/tap is failing to establish connection. Seems to be trying to establish connection to tap pod at port 8089. Tap pod is running and listening on port 8089. I am able to port forward and reach port, also works if I do same install on Minikube.

**APIService status :** v1alpha1.tap.linkerd.io service linkerd-viz/tap status False (FailedDiscoveryCheck)
**Error:** FailedDiscoveryCheck: failing or missing response from https://10.5.122.102:8089/apis/tap.linkerd.io/v1alpha1: Get

Client version: stable-2.14.9
Server version: stable-2.14.9
linkerd/viz : stable-2.14.9

Are there any additional steps for permissions to configure on AWS? I have tried installing with Helm chart as well as linkerd cli, same result. 
 Any help much appreciated. 
Screenshot from ArgoCD install
![image](https://github.com/linkerd/linkerd2/assets/146727480/140820ce-e0ba-4674-a269-a12197f6d1d8)

**Comment by steve0hh** [comment]
For those looking for an answer, this worked for me: https://linkerd.buoyant.io/t/linkerd-viz-tap-faileddiscoverycheck-while-running-on-eks/252

**Reply by kenwjiang** [reply]
thank you!
